ddtrace/__init__.py

```diff
@@ -1,10 +1,13 @@
+from .logger import configure_ddtrace_logger
+
+configure_ddtrace_logger()
```

Comment:
maybe a quick comment that we want this to happen before anything else, so the logger is configured for any potential startup/import logs.

Reply:
I added a comment that says `# noqa: E402 - configure logger for tracer logs` in the latest commit. Is this enough of a note?

Reply:
Marking as resolved.